api/models/app.go

```diff
@@ -645,25 +636,23 @@ func (a *App) RunAttached(process, command, releaseId string, height, width int,
 	time.Sleep(100 * time.Millisecond)
 
 	err = d.StartContainer(res.ID, nil)
-
 	if err != nil {
 		return err
 	}
 
 	err = d.ResizeContainerTTY(res.ID, height, width)
-
 	if err != nil {
-		return err
+		// In some cases, a container might finish and exit by the time ResizeContainerTTY is called.
+		// Resizing the TTY shouldn't cause the call to error out for cases like that.
+		fmt.Printf("fn=RunAttached level=warning msg=\"unable to resize container: %s\"", err)
```

Comment:
I like this. Starting to see a pattern with deciding that some Docker API calls aren't important enough to blow up our API call.